### ISSUE TITLE
Task Create RPC does not return proper information

### ISSUE THREAD

**Issue by ederenn** [golemfactory/clay#3521]
#### Encountered on:

Windows, source GOLEM Version: 0.18.1+dev162.g9ba999a, mainnet

while creating task with not enough ETH available information "not enough funds" is not shown in interface

from golem log:

```
2018-11-06 16:46:32 INFO     golem.ethereum.fundslocker          Locking funds for task: '220c960a-e1db-11e8-aa86-9d5cb6e59b28' price: 0.066667 num: 10
2018-11-06 16:46:32 ERROR    golem.task.rpc                      Cannot create task {'resources': ['C:\\Users\\ederenn\\Desktop\\DiscBoyBlenderPort17_png.blend'], 'taskName': 'DiscBoyBlenderPort17_png.blend', 'relativePath': 'C:\\Users\\ederenn\\Desktop\\DiscBoyBlenderPort17_png.blend', 'name': 'DiscBoyBlenderPort17', 'type': 'Blender', 'compute_on': 'cpu', 'timeout': '1:30:00', 'subtasks_count': 10, 'subtask_timeout': '0:20:00', 'bid': '0.2', 'estimated_memory': 244588544, 'options': {'resolution': [1920, 1080], 'frames': '51-55', 'format': 'PNG', 'compositing': False, 'output_path': 'C:\\Users\\ederenn\\Documents'}, 'concent_enabled': False}: Not enough ETH available. Required: 0.005740, available: 0.002359
```

from electron developers tools:
`comp.task.create (2) ["220c960a-e1db-11e8-aa86-9d5cb6e59b28", null]`

**Comment by ederenn** [comment]
Using Task creation RPC, Golem returns to electron task id but there is no information about "insufficient amount". So electron is not able to show proper modal.
e.g.
```js
comp.task.create -  ["52360212-257d-11e9-8959-3d4a28a162a4", null]
```

from golem logs:

```
2019-01-31 18:25:08 ERROR    golem.task.rpc                      Cannot create task {'resources': ['C:\\Users\\ederenn\\Desktop\\small_shark100.blend'], 'taskName': 'small_shark100.blend', 'relativePath': 'C:\\Users\\ederenn\\Desktop\\small_shark100.blend', 'name': 'small_shark100', 'type': 'Blender', 'compute_on': 'cpu', 'timeout': '0:10:00', 'subtasks_count': 1, 'subtask_timeout': '0:10:00', 'bid': 1, 'estimated_memory': 72171520, 'options': {'resolution': [400, 400], 'frames': '1', 'format': 'PNG', 'compositing': False, 'output_path': 'C:\\Users\\ederenn\\Documents'}, 'concent_enabled': False}: Not enough GNT available. Required: 0.166667, available: 0.000000
```

same situation happens with ETH and GNT

**Comment by ZmijaWA** [comment]
Works fine :)

**Comment by kmazurek** [comment]
Resolved by: #3715
